compass/ocean/tests/ice_shelf_2d/viz.py

```diff
@@ -68,10 +83,80 @@ def run(self):
                                       dims=('yCell', 'nVertLevelsP1'))
         zInterface = zInterface.where(interfaceMask)
 
-        plt.figure(figsize=[24, 12], dpi=100)
-        plt.plot(ds.yCell.values, zMid.values, 'b')
-        plt.plot(ds.yCell.values, zInterface.values, 'k')
-        plt.plot(ds.yCell.values, ds.ssh.values, 'g')
-        plt.plot(ds.yCell.values, -ds.bottomDepth.values, 'g')
+        y = ds.yCell.values / 1.e3
+        plt.figure(figsize=[12, 6], dpi=100)
+        plt.plot(y, zMid.values, 'b', label='zMid')
+        plt.plot(y, zInterface.values, 'k', label='zTop')
+        plt.plot(y, ds.ssh.values, '--g', label='SSH')
+        plt.plot(y, -ds.bottomDepth.values, '--r', label='zBed')
+        plt.xlabel('Distance (km)')
+        plt.ylabel('Depth (m)')
+        plt.legend()
         plt.savefig('vert_grid.png', dpi=200)
         plt.close()
+
+        ds = xarray.open_dataset('initial_state.nc')
+
+        # Plot the time series of max velocity
+        plt.figure(figsize=[12, 6], dpi=100)
+        umax = np.amax(dsOut.velocityX[:, :, 0].values, axis=1)
+        vmax = np.amax(dsOut.velocityY[:, :, 0].values, axis=1)
+        t = dsOut.daysSinceStartOfSim.values
+        time = pd.to_timedelta(t) / 1.e9
+        plt.plot(time, umax, 'k', label='u')
+        plt.plot(time, vmax, 'b', label='v')
+        plt.xlabel('Time (s)')
+        plt.ylabel('Velocity (m/s)')
+        plt.legend()
+        plt.savefig('velocity_max_t.png', dpi=200)
+        plt.close()
+        min_column_thickness = self.config.getfloat(
+            'ice_shelf_2d', 'y1_water_column_thickness')
+
+        figsize = (4, 8)
+        delssh = dsOut.ssh - dsOut.ssh[0, :]
+        s_vmin = np.nanmin(delssh.values)
+        s_vmax = np.nanmax(delssh.values)
+        plotter.plot_horiz_series(delssh, 'delssh', 'delssh', True,
+                                  cmap='cmo.curl',
```

Comment:
```suggestion
        plotter.plot_horiz_series(da=delssh, nameInTitle='delssh', 
                                  prefix='delssh', oceanDomain=True,
                                  cmap='cmo.curl',
```

You can treat `args` as `kwargs` in this way (you just can't leave them out like `kwargs`) and it helps a lot in being about to tell what the args are when you supply constants to them (especially `True` here).

Reply:
One related thing: Maybe we let it be but the viz is written for non-periodic meshes and we're getting weird wrap-around artifacts:
![u_0025](https://github.com/MPAS-Dev/compass/assets/4179064/6be845b1-1f83-47d1-b7d2-39a21878b166)

Reply:
A final note, the date obviously isn't changing since these runs are too short for that.  I guess we're fine with that?

Reply:
I fixed the wrap-around issue in the new polaris script. I'll have to think about whether it is worth migrating those changes. The way I implemented it we need different masks and patches for edge quantities vs. cell quantities.

Reply:
Fine to leave this for Polaris.